tests/testthat/test-derive_merged.R

```diff
@@ -631,3 +631,48 @@ test_that("derive_var_merged_summary Test 23: deprecation warning", {
     class = "lifecycle_error_deprecated"
   )
 })
+
+## Test 24: error if not unique w.r.t the by variables and the order is not specified -
+# case 1, check_type = "error" ----
+test_that("derive_vars_merged Test 24: error if not unique w.r.t the by variables and
+          the order is not specified", {
+  expect_error(
+    actual <- derive_vars_merged(advs,
+      dataset_add = adsl2,
+      by_vars = exprs(STUDYID, USUBJID = ID),
+      order = NULL,
+      check_type = "error"
+    ),
+    regexp = NULL
```

Comment:
Please specify a value for `regexp` which ensures that the expected error is issued. Otherwise, any error is accepted.